dll/win32/shdocvw/mrulist.cpp

```diff
@@ -133,13 +138,15 @@ CMruBase::~CMruBase()
         ::LocalFree(m_pSlots);
         m_pSlots = NULL;
     }
+
+    ::InterlockedDecrement(&SHDOCVW_refCount);
```

Comment:
This is a ref count for the library `shdocvw.dll`.
In the above code of here, the library should be still locked because some object(s) in the library is/are still alive.